relayer/relayer/src/config.rs

```diff
@@ -37,6 +39,10 @@ pub struct Config {
     #[arg(long)]
     pub override_eth_cache: bool,
 
+    /// Optional list of hex encoded request hashes to skip from processing
+    #[arg(long, use_value_delimiter = true, value_delimiter = ',')]
+    pub blacklisted_requests: Option<Vec<H256>>,
```

Comment:
I tried with a set, unfortunately it's missing some traits that make it seamlessly compatible with  `clap`. Similarly empty collections - it's built to work with an Option to support the "optional argument" semantics. going against a framework is usually PITA

Reply:
Yeah, I have some experience with that too. `Option<Vec<...>>` is probably the best here, unless you do some serious hacking.

I'm wondering though, which format of H256 will it accept though? Do you need the `0x` prefix? Probably good to write in the docstring

Reply:
In that case, I think it would be nicer to convert to `HashSet<_>` before passing it further down into the other calls. You're not locked into the types `clap` exposes AFAIK :)